packages/headless/src/controllers/core/generated-answer/headless-core-interactive-citation.ts

```diff
@@ -8,6 +8,8 @@ import {
   type InteractiveResultCoreProps,
 } from '../../core/interactive-result/headless-core-interactive-result.js';
 
+export type {GeneratedAnswerCitation};
```

Comment:
This is already exposed through `headless-generated-answer.ts` is there a reason you need to expose it again here?